modules/@apostrophecms/submitted-draft/index.js

```diff
@@ -105,24 +105,49 @@ module.exports = {
           `${self.__meta.name}:manager`,
           self.pluralLabel,
           {
-            action: 'edit',
+            action: 'publish',
             type: self.name
           },
           {
             component: 'AposSubmittedDraftAdminBarButton'
           }
         );
+        self.apos.adminBar.add(
+          `${self.__meta.name}:manager`,
+          self.pluralLabel,
+          {
+            action: 'edit',
+            type: self.name
+          },
+          {
+            icon: 'AposSubmittedDraftAdminBarIcon',
```

Comment:
I wrote a component to wrap the icon in a v-if test, using the same REST API call as the fancier one for the counter.